cmeutils/structure.py

```diff
@@ -8,6 +8,106 @@
 from cmeutils.geometry import get_plane_normal, angle_between_vectors
 
 
+def angle_distribution(
+        gsd_file, A_name, B_name, C_name, start=0, stop=-1
+):
+    """Returns the bond angle distribution for a given triplet of particles 
+    
+    Parameters
+    ----------
+    gsdfile : str
+        Filename of the GSD trajectory.
+    A_name, B_name, C_name : str
+        Name(s) of particles that form the angle triplet 
+        (found in gsd.hoomd.Snapshot.particles.types)
+        They must be given in the same order as they form the angle
+    start : int
+        Starting frame index for accumulating bond lengths.
+        Negative numbers index from the end. (default 0)
+    stop : int
+        Final frame index for accumulating bond lengths. (default -1)
+
+    Returns
+    -------
+    1-D nump.array 
+        Array of bond angles in degrees
+
+    """
+    angles = []
+    trajectory = gsd.hoomd.open(gsd_file, mode="rb")
+    angle_name = "-".join([A_name, B_name, C_name])
```

Comment:
Might want to also check for C-B-A

Reply:
Good idea, that should be fixed (and tested for) now